### ISSUE TITLE
Should I be able to type in Host IP? (Metroid: Zero Mission)

### ISSUE THREAD

**Issue by Phoenix9108** [TestRunnerSRL/bizhawk-co-op#65]
![Screenshot 2022-10-29 230355](https://user-images.githubusercontent.com/116989230/198860653-8640e288-24b8-452a-8d78-d69dc05018f8.png)

I haven't tested any connectivity yet but the instructions mentioned to put in the IP of whoever's gonna be the host, though I can't type anything into the box. I don't know if it only allows you to when you click "Join Room" because whenever I click "Join Room", it immediately says "Error joining room [Code: 401]". Is this supposed to happen, and is there a fix?

**Comment by josephanimate2021** [comment]
Yes, you can indeed put your own IP address in the Host IP box. You just need to select the (Custom IP) option from the rooms drop down menu and you/your friend should be able to create/join your own room without any problems.
